connectors/src/connectors/google_drive/temporal/workflows.ts

```diff
@@ -160,37 +165,62 @@ export function googleDriveFullSyncWorkflowId(connectorId: ModelId) {
  */
 export async function googleDriveIncrementalSync(
   connectorId: ModelId,
-  dataSourceConfig: DataSourceConfig
+  dataSourceConfig: DataSourceConfig,
+  drivesToSync: DrivesToSyncType | undefined = undefined,
+  nextPageToken: string | undefined = undefined
 ) {
   await syncStarted(connectorId);
```

Comment:
We don't want to execute this again IMO

Reply:
As per irl, it's the same thing for the fullsync workflow, will fix both.

Reply:
🙏